packages/components/src/Select/ListBoxSection.tsx

```diff
@@ -17,15 +16,16 @@ export const ListBoxSection = ({ section, state }: SectionProps) => {
     heading: section.rendered,
     'aria-label': section['aria-label'],
   });
-  const listClassName = useStyles({
-    css: {
-      listStyle: 'none',
-      cursor: 'not-allowed',
-    },
-  });
 
   return (
-    <Box as="li" {...itemProps} className={listClassName}>
+    <Box
+      as="li"
+      {...itemProps}
+      css={{
+        listStyle: 'none',
```

Comment:
Doesn't t make more sense to have the `list-style` defined on the `ul` or `ol` element?

Reply:
Do you mean here https://github.com/marigold-ui/marigold/blob/main/packages/system/src/normalize.ts#:~:text=const%20list%20%3D%20%7B,%7D%20as%20const%3B ?

Reply:
No but in the `<ListBox>` where the `<ListBoxSection>` is used.